app/helpers/guides_helper.rb

```diff
@@ -0,0 +1,9 @@
+module GuidesHelper
+  def book_an_appointment_link?
```

Comment:
In Rails, all helpers are included in all views by default so I think we'll end up with a race condition. However we can configure Rails to only autoload helpers for the specific controller:

```rb
# config/application.rb
config.action_controller.include_all_helpers = false
```